src/rsa.rs

```diff
@@ -432,7 +432,12 @@ enum BIGNUM {}
 
 #[cfg(feature = "rsa_signing")]
 #[allow(non_camel_case_types)]
-enum BN_BLINDING {}
+#[repr(C)]
+struct BN_BLINDING {
+    a: *mut BIGNUM,
+    ai: *mut BIGNUM,
+    counter: u32,
+}
```

Comment:
Usually we avoid doing this, so I'd like to know more about the motivation.
Which test do you expect will need to access these?